cucumber/api/features/authenticate.feature

```diff
@@ -10,93 +10,107 @@ Feature: Exchange a role's API key for a signed authentication token
     And I have host "app"
 
   Scenario: A role's API can be used to authenticate
+    When I save my place in the audit log file for remote
```

Comment:
this looks more like a `Given` to me

Reply:
`When` represents the action you are testing but this is some step that you perform before the test. please change it in the rest of the PR

Reply:
Fixed